app/models/aggregation.js

```diff
@@ -58,6 +58,9 @@ export const RAWS = RawTypes.create();
 export const DISTRIBUTIONS = DistributionTypes.create();
 export const DISTRIBUTION_POINTS = DistributionPointTypes.create();
 
+export const DEFAULT_AGGREGATION_TYPE = 'RAW';
```

Comment:
Make it read from settings?

Reply:
I didn't because I was just replacing what was hardcoded in this file (which I seem to have forgotten to do)